fact_store/app.py

```diff
@@ -1,9 +1,12 @@
 from pybloom import BloomFilter
-from flask import Flask, request, render_template
+from flask import Flask, request, render_template, jsonify
 import pandas as pd
 from fastparquet import write, ParquetFile
 import os
 import datetime
+
+from anomaly_detector.storage.es_storage import ESStorage
+
```

Comment:
ESStorage is used here but I think we will need to make sure that anomaly_detection core is set as the PYTHONPATH env var 
```
PYTHONPATH=${PWD}/../ 
```

Reply:
This is removed.